### PR TITLE
SONAR-6827 Compute remaining measures on views

### PR DESCRIPTION
**Pull request by julienlancelot**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/measure/MeasureRepositoryImpl.java [hunk 1]
```diff
@@ -103,6 +91,11 @@ public void update(Component component, Metric metric, Measure measure) {
     delegate.update(component, metric, measure);
   }
 
+  public Set<Measure> getRawMeasures(Component component, Metric metric) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/computation/measure/MeasureRepositoryImpl.java]: MAJOR Add the "@Override" annotation above this method signature [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)

**Comment by SonarTech** [comment]
SonarQube analysis reported 6 issues:
* MAJOR 6 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by julienlancelot** [comment]
@seb Feedback applied !

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by sns-seb** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 5 issues:
* MAJOR 5 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/input' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/input) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/batch/protocol/output' package (org.codehaus.sonar:sonar-batch-protocol:src/main/gen-java/org/sonar/batch/protocol/output) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonar/db/protobuf' package (org.codehaus.sonar:sonar-db:src/main/gen-java/org/sonar/db/protobuf) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/gen-java/org/sonarqube/ws' package (org.codehaus.sonar:sonar-ws:src/main/gen-java/org/sonarqube/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Pull request merged**